package.json

```diff
@@ -2,8 +2,9 @@
   "name": "immersive",
   "version": "1.0.0",
   "description": "Assignment to create CRUD operations",
-  "main": "server.ts",
+  "main": "server.js",
   "scripts": {
+    "practice": "node server.js",
```

Comment:
Nodemon is already available in the next line...

Reply:
ok